docker-imagebuilder.sh

```diff
@@ -6,40 +6,43 @@ set -x
 TARGETS="${TARGETS:-x86-64}"
 BRANCHES="${BRANCHES:-master}"
 DOCKER_IMAGE="${DOCKER_IMAGE:-openwrt-imagebuilder}"
+DISTROS="alpine debian"
```

Comment:
I'd be in favor to only build one of the two locally per default, and the full range only via CI

Reply:
`DISTROS="${DISTROS:-debian}"`